packages/admin-next/dashboard/src/providers/keybind-provider/hooks.tsx

```diff
@@ -264,6 +264,14 @@ export const useGlobalShortcuts = () => {
       type: "settingShortcut",
       to: "/settings/locations",
     },
+    {
+      keys: {
+        Mac: ["G", ",", "M"],
```

Comment:
question: Would this be more fitting? Typically, the keyboard shortcut is coupled with the resource name
```suggestion
        Mac: ["G", ",", "R"],
```